client/src/theme.ts

```diff
@@ -3,9 +3,10 @@ import { DefaultTheme } from 'styled-components';
 export const myTheme: DefaultTheme = {
     colors: {
         blue: '#2259B1',
-        yellow: '#D4DB2E',
+        lemon: '#D4DB2E',
         red: '#FC6666',
         black: '#11182E',
+        navy_blue: '#214b8d',
```

Comment:
This `navy_blue` color is the same as `dark_blue` on the next line.

Reply:
I did not notice they were the same color code

Reply:
Don't create duplicated color, you can rename it if you want ;)